Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM pcic/geospatial-python
+
+ADD . /app
+WORKDIR /app
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get -yq install postgresql postgresql-contrib postgis postgresql-9.3-postgis-2.1
```

Comment:
Should the above ARG and RUN commands be moved into the `pcic/geospatial-python` package? They are foundational for any image that uses CRMP.